File: 백준/Bronze/2445. 별 찍기 － 8/별 찍기 － 8.py
N = int(input())

for i in range(0, N):
    i += 1
    for x in range(0, i, 1):
        print("*", end="")
    for space in range(0, N * 2 - i * 2, 1):
        print(" ", end="")
    for x in range(0, i, 1):
        print("*", end="")
    print()
for i in range(N - 1, 0, -1):
    for x in range(0, i, 1):
        print("*", end="")
    for space in range(0, N * 2 - i * 2, 1):
        print(" ", end="")
    for x in range(0, i, 1):
        print("*",end= "")
    print()

File: 백준/Bronze/2445. 별 찍기 － 8/별 찍기 － 8.cc
#include <stdio.h>
int main(){
    int N;
    scanf("%d", &N);
    for(int i = 0; i < N;){
        i += 1;
        for(int x = 0; x < i; x++){
            printf("*");
        }
        for(int space = 0; space < N*2-i*2; space++){
            printf(" ");
        }
        for(int x = 0; x < i; x++){
            printf("*");
        }
        printf("\n");
    }
    for(int i = N - 1; i > 0;){
        for(int x = 0; x < i; x++){
            printf("*");
        }
        for(int space = 0; space < N*2-i*2; space++){
            printf(" ");
        }
        for(int x = 0; x < i; x++){
            printf("*");
        }
        printf("\n");
        i -= 1;
    }
}